add_gem_package.sh

```diff
@@ -49,34 +49,74 @@ generate_gem_package() {
 	popd
 }
 
+add_to_all_tito_props() {
+	add_to_tito_props $TITO_TAG
+	if [[ $TITO_TAG == "foreman-nightly-rhel7" ]] || [[ $TITO_TAG == "foreman-nightly-nonscl-rhel7" ]] ; then
```

Comment:
should this instead match `$TITO_TAG == foreman-nightly-*`?

Reply:
Doesn't that mean that if you pass EL8 as a tito tag it gets added twice?

Reply:
Yeah, that'd mean it. But shouldn't the underlying "add" code handle that?

Anyways, that's not blocking, just an idea ;)

Reply:
I thought about that, but I'd prefer to merge this now and improve it if people dislike it. I was mostly annoyed at having to add it manually to additional files and we've seen the same thing in reviews.